VideoWeb/VideoWeb/ClientApp/src/app/services/error.service.spec.ts

```diff
@@ -1,25 +1,36 @@
 import { inject, TestBed } from '@angular/core/testing';
 import { Router } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
+import { AdalService } from 'adal-angular4';
 import { Guid } from 'guid-typescript';
 import { pageUrls } from '../shared/page-url.constants';
 import { MockLogger } from '../testing/mocks/MockLogger';
 import { CallError } from '../waiting-space/models/video-call-models';
+import { ProfileService } from './api/profile.service';
 import { ErrorService } from './error.service';
 import { Logger } from './logging/logger-base';
 import { SessionStorage } from './session-storage';
 import { ErrorMessage } from '../shared/models/error-message';
 
 describe('ErrorService', () => {
     let router: Router;
+    let adalServiceSpy: jasmine.SpyObj<AdalService>;
```

Comment:
could you avoid writing this with the testbed. this really hits the test run and even slower in the pipeline

Reply:
updated to use the mockclasses